auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -280,13 +296,17 @@ export default defineComponent({
         return state.continuationIn?.authorization?.files
       }),
 
-      authorizationDate: computed<string>(() => {
-        return strToPacificDate(state.continuationIn?.authorization?.date)
+      affidavitFileName: computed<string>(() => {
+        return state.continuationIn?.foreignJurisdiction?.affidavitFileName
+      }),
+
+      affidavitFileKey: computed<string>(() => {
+        return state.continuationIn?.foreignJurisdiction?.affidavitFileKey
```

Comment:
Or, you could have a computed that returns `affidavitItem` and does the work from line 215. Up to you.

Reply:
Updated to use affidavitItem